packages/phalanx/test/ViewOfViews/tPhalanxViewOfViews.cpp

```diff
@@ -611,7 +611,10 @@ void testVoV(VoVType& vov, OstreamType& out, bool& success)
   PHX::Device::execution_space().fence();
 
   // Check the results
-  auto c_h = Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace(),c);
+  // FIXME: breaks with new view implementation in 4.7.1
+  //auto c_h = Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace(),c);
```

Comment:
Is this something that still needs to be done as part of this PR?

Reply:
I think we can do this in a follow up. Note that this PR does not change in and of itself behavior, since the default is the old view implementation. But it gets us into a state where we can start testing with Kokkos develop.